src/index.ejs

```diff
@@ -16,7 +16,6 @@
   <script src="cordova.js" defer></script>
   <% } else if (__STACK_ASSETS__) { %>
   {{.CozyClientJS}}
-  {{.CozyBar}}
```

Comment:
`{{.CozyBar}}` was in charge to import the fonts.css so now you have to import it with `{{.CozyFonts}}` (cc: https://github.com/cozy/cozy-drive/pull/2812)